packages/components/filters/src/filter-menu/filter-menu.tsx

```diff
@@ -109,8 +110,23 @@ export const menuStyles = css`
   z-index: 5;
 `;
 
-const menuBodyStyle = css`
+export const menuBodyStyle = css`
   width: 100%;
+
+  /** ensure that body scrolls with overflow now that there is a menu max-height */
+  overflow: hidden auto;
+
+  /** using has selector to add some styles based on whether there is a custom select 3 divs down */
+  :has(div > div > [data-testid='uikit-custom-filters-select']) {
```

Comment:
@ddouglasz this selector might be helpful to enable only scrolling the options menu for the `appearance: filter` work.